agent_payload.proto

```diff
@@ -59,6 +61,28 @@ message EventsPayload {
 	CommonMetadata metadata = 2;
 }
 
+message SketchPayload {
+        message Summary {
```

Comment:
We may want to name this `Percentile/Quantile/something else` instead of `Summary`. If we add top-k or cardinality sketches the name will be confusing.